---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* B.S. in Biology, Massachusetts Institute of Technology, 2014
* Ph.D in Bioinformatics, University of California, Los Angeles, 2018

Research Experience
======
**Arizona State University, Tempe, AZ**
* 2019-Present Analyzing next-generation DNA and RNA sequence data

**University of California, Los Angeles, Los Angeles, CA**
* 2017-2018	Developing an Approximate Bayesian Computational framework to infer the genetic architecture of human complex traits using summary statistics from genome-wide association studies
* 2016-2018	Analyzing genetic diversity on the X chromosome and autosomes to determine whether the demographic history of canines has been sex-biased using whole-genome sequence
* 2015-2017	Developing a likelihood-based approach to understand whether the process of recombination is mutagenic
* 2014-2016	Assessing whether natural selection has reduced neutral divergence at linked sites

**Vertex Pharmaceutical, Boston, MA**
* 2017	Designing a graphical user interphase using RShiny to detect CRISPR mediated insertions and deletions

**Massachusetts Institute of Technology, Cambridge, MA**
* 2013-2014	Investigating amino acid residues that contribute to the specific interaction between the bacterial toxin and antitoxin proteins
* 2012-2013	Conducting microbiology experiments to identify and validate that DnaN, a replicative beta clamp, to be the cellular target of a bacterial toxin protein, SocB
* 2011-2012	Conducting swelling experiments to test the swelling property of hydrogels made with different molecular weights.

**National Institute of Health, Bethesda, MD**
* 2010-2011	Examining the effects of a tumor suppressor gene, CASZ1, on the cell cycle progression.

Publications
======
7. **Phung, T.N.**, Wayne, R.K., Sayres, M.A.W., and Lohmueller, K.E. (2018). Complex patterns of sex-biased demography in canines. BioRxiv 362731.
6. Webster, T.H., Couse, M., Grande, B.M., Karlins, E., **Phung, T.**, Richmond, P.A., Whitford, W., and Sayres, M.A.W. (2018). Identifying, understanding, and correcting technical biases on the sex chromosomes in next-generation sequencing data. BioRxiv 346940.
5. Beichman, A.C., **Phung, T.N.**, and Lohmueller, K.E. (2017). Comparison of Single Genome and Allele Frequency Data Reveals Discordant Demographic Histories. G3: Genes, Genomes, Genetics 7, 3605–3620.
4. **Phung, T.N.**, Huber, C.D., and Lohmueller, K.E. (2016). Determining the Effect of Natural Selection on Linked Neutral Divergence across Species. PLOS Genet 12, e1006199.
3. Aakre, C.D., Herrou, J., **Phung, T.N.**, Perchuk, B.S., Crosson, S., and Laub, M.T. (2015). Evolving New Protein-Protein Interaction Specificity through Promiscuous Intermediates. Cell.
2. Aakre, C.D., **Phung, T.N.**, Huang, D., and Laub, M.T. (2013). A Bacterial Toxin Inhibits DNA Replication Elongation through a Direct Interaction with the β Sliding Clamp. Mol. Cell.
1. Liu, Z., Rader, J., He, S., **Phung, T.**, and Thiele, C.J. (2013). CASZ1 inhibits cell cycle progression in neuroblastoma by restoring pRb activity. Cell Cycle 12, 2210–2218

Fellowship
======
* 2017		UCLA Biomedical Big Data Training Program Fellowship ($24,000 awarded)
* 2016		UCLA System and Integrative Biology Training Program Fellowship ($23,000 awarded)
* 2015		Honorable Mention Ford Foundation Predoctoral Fellowship 
* 2015 		UCLA System and Integrative Biology Training Program Fellowship ($23,000 awarded)

Talks
======
7. Phung TN, Pasaniuc B, Lohmuller KE. Inference of the mutational size supports the omnigenic model for complex trait. Selected talk at Probabilistic Modelling in Genomics 2018. Cold Spring Harbor, NY, November 2018
 
6. Phung TN, Wilson Sayres MA, Lohmueller KE. Comparing genomic patterns of diversity on the X chromosome versus autosomes in dogs. Selected talk at Evolutionary Genomics of Sex 2016. Tempe, AZ, November 2016. 

5. Phung TN, Huber CD, Lohmueller KE. Detecting mutagenic recombination using genome-wide divergence data. Computational Genomics Summer Institute, Los Angeles, CA, July 2016.

4. Phung TN, Huber CD, Lohmueller KE. Examining the effects of natural selection on linked neutral divergence. Selected talk at The Allied Genetics Conference 2018. Orlando, FL, July 2016.

3. Phung TN, Huber CD, Lohmueller KE. A method to detect mutagenic recombination using genome-wide divergence data. Interdepartmental Ph.D. Program in Bioinformatics Retreat. Big Bear Lake, CA, May 2016.

2. Phung TN, Huber CD, Lohmueller KE. Why is genetic variation positively correlated with recombination? Interdepartmental Ph.D. Program in Bioinformatics Retreat. Catalina, CA, May 2015.

1. Phung TN, Aakre C, Laub M. A novel type II toxin-antitoxin system in Caulobacter crescentus blocks replication progression through a direct interaction with DnaN. Selected plenary talk at National Collegiate Research Conference 2013. Cambridge, MA, January 2013. 
