---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* B.S. in Biology, Massachusetts Institute of Technology, 2014
* Ph.D in Bioinformatics, University of California, Los Angeles, 2018

Work Experience
======
**Vrije Universiteit Amsterdam, the Netherlands** (2022 - Present)
* Generating a cell type atlas to investigate cell types that are relevant to traits and diseases by linking single-cell transcriptomics datasets and genetic variants from genome-wide association studies (GWAS)
* Incorporating a gene prioritization to FUMA, a web-platform that allows researchers all over the world to functionally annotate summary statistics from genome-wide association studies, to allow researchers with little bioinformatics skills and/or computational resources to explore their GWAS results

**Xbiome, Boston, MA** (2021 - 2022)
* Developed a bioinformatics pipeline to identify tumor neoantigens from paired tumor-normal WES data from cancer patients
* Developed a bioinformatics pipeline to map tumor neoantigens to the patient’s microbiome to identify bacterial antigens that interact with the immune system
* Applied pipelines that I developed to analyze cancer datasets to study neoantigendiversity, bacterial antigen diversity and association to immunotherapy responses

**Ambry Genetics, Aliso Viejo, CA** (2021 - 2021)
* Developed bioinformatics pipelines to process NGS data from raw BCL files to fastq files and to identify variants from NGS data
* Collaborated R&D scientists within Ambry and with scientists from Sanofi to troubleshoot failed sequencing runs such as analyzing contamination, on target rate, or DNA fragment sizes

**Arizona State University, Tempe, AZ** (2019 - 2021)
* **Project 1****:** Comparing patterns of X-chromosome inactivation between the placenta and adult tissues
  *  Processed whole transcriptome sequence data and whole exome sequence data of 30 placentas from raw fastq files to mapped bam files and variant-call-format to analyze allele specific expression
  *  Wrote scripts in Python to infer whether a gene on the X chromosome escapes X-inactivation and compared between the placenta and adult tissues in the Genotype-Tissue Expression dataset
  *  Wrote an Rshiny application (https://tanyaphung.shinyapps.io/X-inactivation/) to visualize whether a gene escape X-inactivation
    
*  **Project 2:** Analyzing mutational landscape and neoantigens landscape in cancer samples
  *  Developed a pipeline to genotype somatic mutations in cancer patients by incorporating existing softwares to identify overlap in mutations identified in at least two softwares
  *  Built upon existing softwares to identify and prioritize neoantigens to be validated as potential vaccines for cancer treatment
  *  Collaborated with the Mayo Clinic to analyze heterogeneity in mutational landscape and neoantigen diversity from four different cancers of the same Lynch syndrome patient and from different breast cancers
    
*  **Project 3:** Identifying genes that are differentially expressed in idiopathic subglottic stenosis
  * Processed whole transcriptome sequence data from the trachea (control tissue) and the subglottic (diseased tissue) tissues of 12 patients to identify differentially expressed genes
  * Collaborated with clinicians at the Mayo Clinic and communicated scientific findings

**University of California, Los Angeles, Los Angeles, CA** (2014 - 2018)
* Developing an Approximate Bayesian Computational framework to infer the genetic architecture of human complex traits using summary statistics from genome-wide association studies
* Analyzing genetic diversity on the X chromosome and autosomes to determine whether the demographic history of canines has been sex-biased using whole-genome sequence
* Developing a likelihood-based approach to understand whether the process of recombination is mutagenic
* Assessing whether natural selection has reduced neutral divergence at linked sites

**Vertex Pharmaceutical, Boston, MA** (2017)
* Designing a graphical user interphase using RShiny to detect CRISPR mediated insertions and deletions

**Massachusetts Institute of Technology, Cambridge, MA** (2011 - 2014)
* Investigating amino acid residues that contribute to the specific interaction between the bacterial toxin and antitoxin proteins
* Conducting microbiology experiments to identify and validate that DnaN, a replicative beta clamp, to be the cellular target of a bacterial toxin protein, SocB
* Conducting swelling experiments to test the swelling property of hydrogels made with different molecular weights.

**National Institute of Health, Bethesda, MD** (2010 - 2011)
* Examining the effects of a tumor suppressor gene, CASZ1, on the cell cycle progression.

Publications
======
13. **Phung, T. N.**, Olney, K. C., Pinto, B. J., Silasi, M., Perley, L., O’Bryan, J. P., Kliman, H. J., & Wilson, M. L. (2022). X chromosome inactivation in the human placenta is patchy and distinct from adult tissues. Human Genetics and Genomics Advances, 3(3), 100121. https://doi.org/10.1016/j.xhgg.2022.100121
12. Borden E.S., **Phung T.N.** et al (2021) Penalized regression analysis identifies features of the peptide: MHC class I interaction and mRNA expression as key to prioritizing neoantigen immunogenicity. The Journal of Immunology.
11. **Phung T.N.** et al. (2021) Unique evolutionary trajectories of breast cancers with distinct genomic and spatial heterogeneity. Scientific reports, Volume 11, Issue 1.
10. **Phung T.N.** et al. (2020) Unique genomic and neoepitope landscapes across tumors: a study across time, tissues, and space within a single lynch syndrome patient. Scientific reports, Volume 10, Issue 1.
9. Grujic O., **Phung T.N.** , Kwon S.B., Arneson A., Lee Y., Lohmueller K.E., and Ernst J. (2020). Identification and characterization of constrained non-exonic bases lacking predictive epigenomic and transcription factor binding annotations. Nature Communications, 11(1). https://doi.org/10.1038/s41467-020-19962-9
8. Borden, E.S., Kang, P., Natri, H.M., **Phung, T.N.**, Wilson, M.A., Buetow, K.H., Hastings, K.T. (2019). Neoantigen fitness model predicts lower immune recognition of cutaneous squamous cell carcinomas than actinic keratoses. Frontiers in immunology, Volume 10, Issue 29.
7. **Phung, T.N.**, Wayne, R.K., Sayres, M.A.W., and Lohmueller, K.E. (2018). Complex patterns of sex-biased demography in canines. BioRxiv 362731.
6. Webster, T.H., Couse, M., Grande, B.M., Karlins, E., **Phung, T.**, Richmond, P.A., Whitford, W., and Sayres, M.A.W. (2018). Identifying, understanding, and correcting technical biases on the sex chromosomes in next-generation sequencing data. BioRxiv 346940.
5. Beichman, A.C., **Phung, T.N.**, and Lohmueller, K.E. (2017). Comparison of Single Genome and Allele Frequency Data Reveals Discordant Demographic Histories. G3: Genes, Genomes, Genetics 7, 3605–3620.
4. **Phung, T.N.**, Huber, C.D., and Lohmueller, K.E. (2016). Determining the Effect of Natural Selection on Linked Neutral Divergence across Species. PLOS Genet 12, e1006199.
3. Aakre, C.D., Herrou, J., **Phung, T.N.**, Perchuk, B.S., Crosson, S., and Laub, M.T. (2015). Evolving New Protein-Protein Interaction Specificity through Promiscuous Intermediates. Cell.
2. Aakre, C.D., **Phung, T.N.**, Huang, D., and Laub, M.T. (2013). A Bacterial Toxin Inhibits DNA Replication Elongation through a Direct Interaction with the β Sliding Clamp. Mol. Cell.
1. Liu, Z., Rader, J., He, S., **Phung, T.**, and Thiele, C.J. (2013). CASZ1 inhibits cell cycle progression in neuroblastoma by restoring pRb activity. Cell Cycle 12, 2210–2218

Grants and Fellowships
======
* 2023    I was awarded €55,000 from Amsterdam Neuroscience to explore how to best incorporate multi-omics data to prioritize genes for functional follow-up
* 2017		UCLA Biomedical Big Data Training Program Fellowship ($24,000 awarded)
* 2016		UCLA System and Integrative Biology Training Program Fellowship ($23,000 awarded)
* 2015		Honorable Mention Ford Foundation Predoctoral Fellowship 
* 2015 		UCLA System and Integrative Biology Training Program Fellowship ($23,000 awarded)

Talks
======
7. Phung TN, Pasaniuc B, Lohmuller KE. Inference of the mutational size supports the omnigenic model for complex trait. Selected talk at Probabilistic Modelling in Genomics 2018. Cold Spring Harbor, NY, November 2018
 
6. Phung TN, Wilson Sayres MA, Lohmueller KE. Comparing genomic patterns of diversity on the X chromosome versus autosomes in dogs. Selected talk at Evolutionary Genomics of Sex 2016. Tempe, AZ, November 2016. 

5. Phung TN, Huber CD, Lohmueller KE. Detecting mutagenic recombination using genome-wide divergence data. Computational Genomics Summer Institute, Los Angeles, CA, July 2016.

4. Phung TN, Huber CD, Lohmueller KE. Examining the effects of natural selection on linked neutral divergence. Selected talk at The Allied Genetics Conference 2018. Orlando, FL, July 2016.

3. Phung TN, Huber CD, Lohmueller KE. A method to detect mutagenic recombination using genome-wide divergence data. Interdepartmental Ph.D. Program in Bioinformatics Retreat. Big Bear Lake, CA, May 2016.

2. Phung TN, Huber CD, Lohmueller KE. Why is genetic variation positively correlated with recombination? Interdepartmental Ph.D. Program in Bioinformatics Retreat. Catalina, CA, May 2015.

1. Phung TN, Aakre C, Laub M. A novel type II toxin-antitoxin system in Caulobacter crescentus blocks replication progression through a direct interaction with DnaN. Selected plenary talk at National Collegiate Research Conference 2013. Cambridge, MA, January 2013. 
