---
layout: archive
title: "Coding portfolio"
permalink: /portfolio/
author_profile: true
---

Below describes my coding projects and links to my Github repository

## 1. Develop a tool called `vcfhelper` in Python to aid with processing variant-calling-format (VCF) files. 
* `vcfhelper` extracts any annotations of interest of the VCF file which can then be used to plot for visualization
* [Github link](https://github.com/tanyaphung/vcfhelper)

## 2. Develop workflows for running analyses on UKB-RAP
* [Github link](https://github.com/tanyaphung/ukb_rap_workflows)

## 3. Parse GTEx dataset
* [Github link](https://github.com/tanyaphung/ParseGtex)

## 4. rshiny app projects
- Over the years, I have developed applications in Rshiny to share results from my research. 
- [Github repo](https://github.com/tanyaphung/rshiny_tutorial) where I share some notes on Rshiny development. 
- I have published the following Rshiny apps: 
1. NIPT analyses: 
- [https://tanyaphung.shinyapps.io/nipt_analyses/](https://tanyaphung.shinyapps.io/nipt_analyses/)
- [Github link](https://github.com/tanyaphung/nipt_analyses)

2. X chromosome inactivation: 
- [https://tanyaphung.shinyapps.io/X-inactivation/](https://tanyaphung.shinyapps.io/X-inactivation/)

3. scRNAseq viewer: 
- [https://tanyaphung.shinyapps.io/scrna_category_rshiny/](https://tanyaphung.shinyapps.io/scrna_category_rshiny/)

## 5. Command-line version of FUMA Cell type
* [Github link](https://github.com/tanyaphung/FUMA_Celltype_cmd)

## 6. RNAseq analyses
* [Github link](https://github.com/tanyaphung/rnaseq_analysis_scripts)

## 7. Webtool development
- As part of my work at the VU, I am maintaining and updating a webtool called FUMA. Here are some of the tasks I have contributed to: 
* Implemented a functionality to delete jobs by administrator: [Github link](https://github.com/vufuma/FUMA-webapp/pull/39/files)
* Updated the GUI for one of the pages: [Github link](https://github.com/vufuma/FUMA-webapp/pull/5/files)
